daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -39,30 +31,69 @@ The following Dapr control plane deployments are optional:
+For more details on configuring resource in Kubernetes see [Assign Memory Resources to Containers and Pods](https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/) and [Assign CPU Resources to Containers and Pods](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/)
+
+Example settings for the dapr sidecar in a production-ready setup:
 
 | CPU | Memory |
 |-----|--------|
-| Limit: 4, Request: 100m | Limit: 4000Mi, Request: 250Mi
+| Limit: 300m, Request: 100m | Limit: 1000Mi, Request: 250Mi
 
 *Note: Since Dapr is intended to do much of the I/O heavy lifting for your app, it's expected that the resources given to Dapr enable you to drastically reduce the resource allocations for the application*
 
-The CPU and memory limits above account for the fact that Dapr is intended to do a lot of high performant I/O bound operations. Based on your app needs, you can increase or decrease those limits accordingly.
+The CPU and memory limits above account for the fact that Dapr is intended to a high number of I/O bound operations. It is strongly recommended that you use a tool monitoring tool to baseline the sidecar (and app) containers and tune these settings based on those baselines.
 
 ## Deploying Dapr with Helm
 
-When deploying to a production cluster, it's recommended to use Helm. The Dapr CLI installation into a Kubernetes cluster is for a development and test only setup.
+When deploying to a production cluster, it's recommended to use Helm. Although the Dapr CLI can install Dapr onto a Kubernetes cluster, it is intended for use in dev/test scenarios.
 You can find information [here]({{< ref "install-dapr-selfhost.md#using-helm-advanced" >}}) on how to deploy Dapr using Helm.
 
-When deploying Dapr in a production-ready configuration, it's recommended to deploy with a highly available configuration of the control plane:
+When deploying Dapr in a production-ready configuration, it's recommended to deploy with a highly available configuration of the control plane. It is recommended to create a values file instead of specifying parameters on the command-line. This file should be checked in to source control so that you can track changes made to it.
+
+
+
+For a full list of all available options you can set in the values file (or by using the `--set` command-line option), see https://github.com/dapr/dapr/blob/master/charts/dapr/README.md.
 
 ```bash
-helm install dapr dapr/dapr --version=<Dapr chart version> --namespace dapr-system --set global.ha.enabled=true
+# add/update the helm repo
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+
+# See which chart versions are available
+helm search repo dapr --devel --versions
+
+# create a values file to store variables
+touch values.yml
+cat << EOF >> values.yml
+global.ha.enabled: true
+
+EOF
+
+# run install/upgrade
+helm upgrade --install dapr dapr/dapr \
```

Comment:
do we need the `--install` flag here? I tested upgrades and they went fine without it

Reply:
`helm upgrade --install` is a catch all, similar to `kubectl apply`: "make sure its there by upgrading or installing". But I'll remove `--install` to avoid confusion.

Reply:
fixed.